target/linux/ramips/dts/RB750Gr3.dts

```diff
@@ -7,13 +7,13 @@
 
```

Comment:
@arapov As you're touching DTS, could you please add proper SPDX license identifier, something like `SPDX-License-Identifier: GPL-2.0-or-later OR MIT` for details please see 1. in [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist).

@ndoo As the original author, are you ok with this license?

Reply:
@ynezz I indeed all in favour of GPL2+. If I don't hear from @ndoo, I'll apply the License-Identifier, though it is @ndoo's right to evoke it.

Reply:
Wont work this way. You are **only allowed** to change the licence if all contributor to this file agree. And there are quite a lot: https://github.com/openwrt/openwrt/commits/master/target/linux/ramips/dts/RB750Gr3.dts

Reply:
@mkresin Ok, I see.

Reply:
@diizzyy https://github.com/openwrt/openwrt/pull/1668#issuecomment-449692907

Reply:
@ynezz Cool stuff! Although I will reach out to folks who contributed to this dts early 2019 and update it with license.

Reply:
@ynezz I've got the ACKs! >> https://github.com/openwrt/openwrt/pull/1747

Reply:
@arapov Great, thanks a lot!